common/luaobject.h

```diff
@@ -195,6 +195,16 @@ int luaA_object_emit_signal_simple(lua_State *);
         return 1; \
     }
 
+#define LUA_OBJECT_EXPORT_PROPERTY2(pfx, type, field, pusher, empty_value) \
```

Comment:
sorry for bikeshedding proposal but mb better name it LUA_OBJECT_EXPORT_PROPERTY_WFALLBACK or somehow like that?